### PR TITLE
Updates to instructions in HoC progression

### PR DESCRIPTION
**Pull request by fisher-alice**
Levelbuilder editing is currently not working so these are updates to instructions from @katiejofr based on Karim's feedback. In anticipation of upcoming change to hint bubbles in instructions so that they will have blue border, this PR also adds blue lightbulb images to hints.

 I checked locally to make sure changes look as expected.
Screencast video: I show validation feedback only on level 2.

https://github.com/code-dot-org/code-dot-org/assets/107423305/e80213e1-3dcd-4a23-97c1-864eeb4d66ec

## Links
[Slack thread](https://codedotorg.slack.com/archives/C05DMS18MEX/p1699925505783849)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jmkulwik — COMMENTED** [review]
Has anyone synced with the international team to understand whether we can get these translated?

**Comment by fisher-alice** [comment]
> Has anyone synced with the international team to understand whether we can get these translated?

Yes - @samantha-code has.

File: dashboard/config/levels/custom/dance/dance_ai_generate_effect.level [hunk 1]
```diff
@@ -53,8 +53,8 @@
...
-    "short_instructions": "**A.I. is here to find the best effect for your dance party!**\r\nConnect the `AI generate effect` block to setup then click on the block to get started.",
+    "long_instructions": "**A.I. can help you create fun visual effects for your dance party!**\r\n\r\n<img src=\"https://images.code.org/4a3bd15e97f6dd79f9a57fc07ad79017-image-1697662602704.png\" style=\" margin-right: 40px;  height: 100px; float: left\">\r\n\r\n- Connect the <code style=\"color:white; background-color:#566064\">AI generate effect</code> block to setup.\r\n\r\n- Click on the block to get started. \r\n",
+    "short_instructions": "**A.I. can help you create fun visual effects for your dance party!**\r\nConnect the `AI generate effect` block to setup. Then click on the block to get started.",
```
Comment (Erin007) [inline dashboard/config/levels/custom/dance/dance_ai_generate_effect.level]: no preference on my end just confirming the inconsistency in A.I. v. AI is expected

**Review by Erin007 — APPROVED** [review]
No concerns from me as the outgoing DOTD on these strings, @fisher-alice  👍

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

File: dashboard/config/levels/custom/dance/dance_ai_generate_effect.level (hunk 1 above)
Comment (fisher-alice) [reply dashboard/config/levels/custom/dance/dance_ai_generate_effect.level]: Thanks! I'm going to check with the team about this.

Comment (fisher-alice) [reply dashboard/config/levels/custom/dance/dance_ai_generate_effect.level]: Thanks so much Erin! We do want consistency so I updated in three levels.

**Pull request merged**